api/src/test/java/org/openmrs/module/mirebalais/wristband/WristbandTemplateTest.java

```diff
@@ -154,7 +154,7 @@ public void testWristBandTemplate() {
         String output = wristbandTemplate.generateWristband(patient, visitLocation);
 
         assertThat(output, containsString("^XA^CI28^MTD^FWB"));
-        assertThat(output, containsString("^FO050,200^FB2150,1,0,L,0^AS^FDHôpital Universitaire de Mirebalais " + df.format(today) + "^FS"));
+        assertThat(output, containsString("^FO050,200^FB2150,1,0,L,0^AS^FDMirebalais " + df.format(today) + "^FS"));
```

Comment:
Is this a user facing change? Something different gets printed on the wristbands? Is that okay?

Reply:
Thanks @brandones ! I asked the same question at standup two days ago. I got no answer yet. :-)

Reply:
Huh. I guess it's hard to imagine this causing a problem in the hospital, since all the wristbands will start printing differently all at the same time once we deploy. But it seems like it would be good to at least notify someone there. Let's bring it up after stand-up.

Reply:
Agreed!  I think this affects the ID cards as well... not too worried about it, but agree that we should let the users know...